vendor/inline-scripts/load-ads.js

```diff
@@ -40,27 +40,23 @@
 		// Note that at this point we don't have geo module
 		// TODO#2: load adProducts* script in parallel
 		if (true) {
-			loadScript(assetUrls.adEngineScript)
-				.then(function () {
-					return loadScript(assetUrls.adProductsScript)
-				})
-				.then(function () {
-					return loadScript(assetUrls.geoScript)
-				})
-				.then(function () {
-					adEngine3Loaded = true;
-					onAdsLoaded();
+			loadScript(assetUrls.adEngineScript, function onEngineLoaded() {
```

Comment:
What was wrong with promises? ;-)

Reply:
This file is not transpiled. It's for backwards compatibility.